go.mod

```diff
@@ -335,10 +335,11 @@ require (
 require (
 	github.com/filecoin-project/boost-gfm v1.26.7
 	github.com/filecoin-project/boost-graphsync v0.13.6
+	github.com/filecoin-project/go-data-segment v0.0.0-20230605095649-5d01fdd3e4a1
```

Comment:
Can we point at a release version of go-data-segment?

Reply:
this is the release version i think:
https://github.com/filecoin-project/go-data-segment/releases/tag/0.0.1

because the tag is '0.0.1' and not 'v0.0.1' go.mod uses the commit hash like this to describe it.

cc @Kubuxu

Reply:
Hrm, I will fix it.

Reply:
Released: https://github.com/filecoin-project/go-data-segment/releases/tag/v0.0.1